README.md

```diff
@@ -26,23 +26,42 @@ This works similar to the `alias` command:
     # remove an alias
     $ brew unalias i
 
-Note that some commands are reserved and can’t be aliased (Homebrew core
+> **Note:** Some commands are reserved and can’t be aliased (Homebrew core
```

Comment:
They are meant to represent a quote in Markdown and GitHub renders them in a `blockquote`.